app/frontend/Shared/WebView.svelte

```diff
@@ -52,8 +52,34 @@
   function haveWebView () {
     webviewE.addEventListener("dom-ready", () => {
       dispatch("webview", webviewE);
+      webviewE.addEventListener("did-finish-load", resizeWebview());
     }, { once: true });
   }
+
+  const heightBuffer = 20;
+  async function resizeWebview () {
+    try {
+      const dimensions = await webviewE.executeJavaScript(`
+        document.body.style.minHeight = "0px";
+        document.body.style.height = "fit-content";
+        document.body.style.width = "fit-content";
+        new Promise(resolve => {
+          const observer = new ResizeObserver(entries => {
+            const contentRect = entries[0].contentRect;
+            resolve({ 
```

Comment:
There's a trailing space here. Please make sure to remove them before submitting a patch.